src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Configurations/ICrazyPriceLogger.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Extensions.Hosting;
+
+namespace Exadel.CrazyPrice.Common.Configurations
```

Comment:
Are you sure that we need this file? In a case of 2 loggers no sense to create extra interface

Reply:
Removed.